llvmlite/tests/test_binding.py

```diff
@@ -398,7 +398,7 @@ def test_set_option(self):
     def test_version(self):
         major, minor, patch = llvm.llvm_version_info
         # one of these can be valid
-        valid = [(10, 0), (9, 0)]
+        valid = [(11, 1)]
```

Comment:
```suggestion
        valid = [(11, 1)]
```
same query RE strictness of minor version.

Reply:
sure